yoti-sdk-spring-boot-auto-config/src/main/java/com/yoti/api/spring/YotiClientProperties.java

```diff
@@ -63,21 +86,25 @@ public boolean equals(final Object o) {
         YotiClientProperties that = (YotiClientProperties) o;
 
         if (clientSdkId != null ? !clientSdkId.equals(that.clientSdkId) : that.clientSdkId != null) return false;
+        if (applicationId != null ? !applicationId.equals(that.applicationId) : that.applicationId != null) return false;
         return accessSecurityKey != null ? accessSecurityKey.equals(that.accessSecurityKey) : that.accessSecurityKey == null;
 
     }
 
     @Override
     public int hashCode() {
+        int prime = 31;
         int result = clientSdkId != null ? clientSdkId.hashCode() : 0;
-        result = 31 * result + (accessSecurityKey != null ? accessSecurityKey.hashCode() : 0);
+        result = prime * result + (applicationId != null ? applicationId.hashCode() : 0);
```

Comment:
Couldn't see any usages of this class, and 900+ usages of the base class, so not really sure where this is used. 
I used the same format that [/blob/master/yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/Receipt.java](https://github.com/getyoti/yoti-java-sdk/blob/master/yoti-sdk-impl/src/main/java/com/yoti/api/client/spi/remote/call/Receipt.java#L160) uses

Reply:
You really should let your IDE manage the generation of the hashCode method.